include/xtensor/xscalar.hpp

```diff
@@ -714,14 +712,14 @@ namespace xt
     template <layout_type L>
     inline auto xscalar<CT>::rbegin() noexcept -> reverse_iterator
     {
-        return reverse_storage_iterator(end());
+        return reverse_iterator(end());
     }
 
     template <class CT>
     template <layout_type L>
     inline auto xscalar<CT>::rend() noexcept -> reverse_iterator
     {
-        return reverse_storage_iterator(begin());
+        return reverse_iterator(begin());
```

Comment:
storage iterators did not exist on xscalar. It seems these functions are not used in any test.